exporter/SynthesisFusionAddin/src/Logging.py

```diff
@@ -30,14 +30,19 @@ def cleanupHandlers(self) -> None:
 def setupLogger() -> SynthesisLogger:
     now = datetime.now().strftime("%H-%M-%S")
     today = date.today()
-    logFileFolder = getOSPath(f"{pathlib.Path(__file__).parent.parent}", "logs")
-    logFiles = [os.path.join(logFileFolder, file) for file in os.listdir(logFileFolder) if file.endswith(".log")]
-    logFiles.sort()
-    if len(logFiles) >= MAX_LOG_FILES_TO_KEEP:
-        for file in logFiles[: len(logFiles) - MAX_LOG_FILES_TO_KEEP]:
-            os.remove(file)
-
-    logFileName = f"{logFileFolder}{getOSPath(f'{INTERNAL_ID}-{today}-{now}.log')}"
+    if not IS_RELEASE:
+        logFileFolder = makeDirectories(os.path.join(f"{pathlib.Path(__file__).parent.parent}", "logs"))
+    else:
+        logFileFolder = makeDirectories(os.path.join(SUPPORT_PATH, "Logs"))
+
+    if not IS_RELEASE:
+        logFiles = [os.path.join(logFileFolder, file) for file in os.listdir(logFileFolder) if file.endswith(".log")]
+        logFiles.sort()
+        if len(logFiles) >= MAX_LOG_FILES_TO_KEEP:
+            for file in logFiles[: len(logFiles) - MAX_LOG_FILES_TO_KEEP]:
+                os.remove(file)
```

Comment:
Is this split into multiple if statements for readability?

Reply:
More or less, I would be open to other suggestions if you think you have a better way to format this.

Reply:
I mean I wouldn't have an issue with:

```suggestion
    if not IS_RELEASE:
        logFileFolder = makeDirectories(os.path.join(f"{pathlib.Path(__file__).parent.parent}", "logs"))

        logFiles = [os.path.join(logFileFolder, file) for file in os.listdir(logFileFolder) if file.endswith(".log")]
        logFiles.sort()
        if len(logFiles) >= MAX_LOG_FILES_TO_KEEP:
            for file in logFiles[: len(logFiles) - MAX_LOG_FILES_TO_KEEP]:
                os.remove(file)
    else:
        logFileFolder = makeDirectories(os.path.join(SUPPORT_PATH, "Logs"))    
```